Core/Exchange.php

```diff
@@ -20,6 +20,8 @@ class Exchange implements SerializableInterface
     const HEADER_PARENT_EXCHANGE = 'parent_exchange';
     const HEADER_HANDLER = 'handler';
     const HEADER_FROM = 'from';
+    const HEADER_METHOD = 'method';
+    const HEADER_STATUS = 'status';
```

Comment:
These properties can be removed from here and be added in platform if there are needed just for the callback success or failure